lib/blacklight/request_builders.rb

```diff
@@ -51,6 +51,14 @@ def solr_search_params_logic= logic
       deprecation_deprecate :solr_search_params_logic=
     end
 
+    def search_builder_class
```

Comment:
Perhaps the default `search_params_logic` and `search_builder_class` can go into config, but I would like to retain the ability to pass the processor chain to `search_builder` so that I can use different chains for different methods.  This would help me simplify code like this: https://github.com/curationexperts/alexandria-v2/blob/5dc416a45effae25bf3473224bc84f4740ec8770/app/controllers/collections_controller.rb#L52-L61